src/pages/dashboardGroupDetail/GroupDetailPage.tsx

```diff
@@ -117,7 +117,18 @@ function GroupDetailPage({ partyGroup, party, productsMap, productsRolesMap }: P
         <Grid item xs={4}>
           <Box display="flex">
             <Box>
-              <Typography variant="h4">{partyGroup.name}</Typography>
+              <Typography
+                variant="h4"
+                sx={{
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
```

Comment:
apply the same behaviour used in Users:
sx={{
                width: '100%',
                wordWrap: 'break-word',
              }}